changelog/21.0/21.0.0/summary.md

```diff
@@ -137,3 +138,6 @@ Currently many of the configuration options for VReplication Workflows are vttab
 requires restarts of vttablets. We now allow these to be overridden while creating a workflow or dynamically once
 the workflow is in progress. See https://github.com/vitessio/vitess/pull/16583 for details.
 
+### <a id="errant-gtid-metric"/>Errant GTIDs Count Metric
+A new metric called `ErrantGTIDCounts` has been added to the `VTOrc` component. 
```

Comment:
A count[er] stat is an ever increasing metric that indicates the number of times X has occurred. So I would assume this is the number of times it's had an errant GTID. I would suggest calling this ~ `CurrentErrantGTIDCount` (which is a gauge vs a counter).

Reply:
Fixed